solr/core/src/java/org/apache/solr/core/CoreContainer.java

```diff
@@ -423,6 +424,20 @@ public CoreContainer(NodeConfig config, CoresLocator locator, boolean asyncSolrC
     this.allowListUrlChecker = AllowListUrlChecker.create(config);
   }
 
+  private SolrCores newSolrCores(NodeConfig config) {
```

Comment:
does this need the 'config' param? it could use the already existing 'this.cfg' (similar to the loader)?